docs/import-data/overview.mdx

```diff
@@ -76,7 +76,7 @@ export default function ConnectToMemgraph() {
             <li>
               <b>
                 <a href="#kafka">
-                  How to import streaming data from Kafka into Memgraph?
+                  How to import streaming data from Kafka or Redpanda into Memgraph?
```

Comment:
I guess we don't have a how-to for Pulsar, but it is a valid method of import, so i think it should be written here

Reply:
again opening a pandora's box! How are these guides different from the how-to guides or tutorials? 

I think we should just have 1 page called Import data, and there are all the ways you can import data and the how to guides, tutorials or whatever referencing that particular way of importing data is linked...